test/ng/compileSpec.js

```diff
@@ -11028,91 +11083,110 @@ describe('$compile', function() {
     );
   });
 
-  describe('*[src] context requirement', function() {
-
-    it('should NOT require trusted values for img src', inject(function($rootScope, $compile, $sce) {
-      element = $compile('<img src="{{testUrl}}"></img>')($rootScope);
-      $rootScope.testUrl = 'http://example.com/image.png';
-      $rootScope.$digest();
-      expect(element.attr('src')).toEqual('http://example.com/image.png');
-      // But it should accept trusted values anyway.
-      $rootScope.testUrl = $sce.trustAsUrl('http://example.com/image2.png');
-      $rootScope.$digest();
-      expect(element.attr('src')).toEqual('http://example.com/image2.png');
-    }));
-
+  ['img', 'audio', 'video'].forEach(function(tag) {
     // Support: IE 9 only
-    // IE9 rejects the video / audio tag with "Error: Not implemented" and the source tag with
-    // "Unable to get value of the property 'childNodes': object is null or undefined"
-    if (msie !== 9) {
-      they('should NOT require trusted values for $prop src', ['video', 'audio'],
-      function(tag) {
-        inject(function($rootScope, $compile, $sce) {
+    // IE9 rejects the `video` / `audio` tags with "Error: Not implemented"
+    if (msie !== 9 || tag === 'img') {
+      describe(tag + '[src] context requirement', function() {
+        it('should NOT require trusted values for whitelisted URIs', inject(function($rootScope, $compile) {
           element = $compile('<' + tag + ' src="{{testUrl}}"></' + tag + '>')($rootScope);
-          $rootScope.testUrl = 'http://example.com/image.mp4';
+          $rootScope.testUrl = 'http://example.com/image.mp4'; // `http` is whitelisted
           $rootScope.$digest();
           expect(element.attr('src')).toEqual('http://example.com/image.mp4');
+        }));
 
-          // But it should accept trusted values anyway.
-          $rootScope.testUrl = $sce.trustAsUrl('http://example.com/image2.mp4');
+        it('should accept trusted values', inject(function($rootScope, $compile, $sce) {
+          // As a MEDIA_URL URL
+          element = $compile('<' + tag + ' src="{{testUrl}}"></' + tag + '>')($rootScope);
```

Comment:
I think the CI errors you get are due to some browser protections on assigning javascript: to images... Those are annoying, I remember hitting these on the IE family.

Reply:
Right, I wondered about that. I'll remove the `javascript:`s